composer.json

```diff
@@ -31,7 +31,6 @@
 	"require-dev": {
 		"friendsofphp/php-cs-fixer": "^3.14",
 		"phpunit/phpunit": "^9.6",
-		"roave/security-advisories": "dev-latest",
```

Comment:
This clubdirectory version is TYPO3 12 compatible only. So, it's a supported version. Why removing that package?